architecture/adr-list/adr-002-identity-transactions.md

```diff
@@ -0,0 +1,272 @@
+    {
+      "id": "N22KY2Dyvmuu2PyyqSFKue#z6MkiukuAuQAE8ozxvmahnQGzApvtW7KT5XXKfojjwbdEomY",
+      "type": "Ed25519VerificationKey2020", // external (property value)
+      "controller": "N22KY2Dyvmuu2PyyqSFKue",
+      "publicKeyMultibase": "z5TVraf9itbKXrRvt2DSS95Gw4vqU3CHAdetoufdcKazA"
+    }
+  ]
+ }
+```
+
+#### Verification Method
+
+1. **`id`** (string): A string with format `<DIDDoc-id>#<key-alias`
+2. **`controller`**: A list of base58-encoded identifier strings.
+3. **`type`** (string)
+4. **`publicKeyJwk`** (`map[string,string]`, optional): A map representing a JSON Web Key that conforms to [RFC7517](https://tools.ietf.org/html/rfc7517). See definition of `publicKeyJwk` for additional constraints.
+5. **`publicKeyMultibase`** (optional): A base58-encoded string that conforms to a [MULTIBASE](https://datatracker.ietf.org/doc/html/draft-multiformats-multibase-03) encoded public key.
+
+**Example:**
+
+```javascript
+{
+  "id": "N22KY2Dyvmuu2PyyqSFKue#key-0",
+  "type": "JsonWebKey2020",
+  "controller": "N22KY2Dyvmuu2PyyqSFKue",
+  "publicKeyJwk": {
+    "kty": "OKP", // external (property name)
+    "crv": "Ed25519", // external (property name)
+    "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ" // external (property name)
+}
+```
+
+#### Service
+
+1. **`id`** (string): The value of the id property MUST be a URI conforming to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986). A conforming producer MUST NOT produce multiple service entries with the same ID. A conforming consumer MUST produce an error if it detects multiple service entries with the same ID. It has a follow format: `<DIDDoc-id>#<service-alias>`
+2. **`type`** (string): The service type and its associated properties SHOULD be registered in the DID Specification Registries [DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-spec-registries/)
+3. **`serviceEndpoint`** (strings): A string that conforms to the rules of [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) for URIs, a map, or a set composed of a one or more strings that conform to the rules of [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) for URIs and/or maps.
+
+**Example:**
+
+ ```javascript
+"service": [{
+  "id":"N22KY2Dyvmuu2PyyqSFKue#linked-domain",
+  "type": "LinkedDomains",
+  "serviceEndpoint": "https://bar.example.com"
+}]
+```
+
+#### Update `DID`
+
+If there is no DID transaction with the specified DID \(`dest`\), it is considered as a creation request for a new DID.
+
+If there is a DID transaction with the specified DID \(`dest`\), then this is update of existing DID. In this case, we can specify only the values we would like to override. All unspecified values remain the same. E.g., if a key rotation needs to be performed, the owner of the DID needs to send a DID transaction request with `dest`, `verkey` only. `alias` will stay the same.
+
+**Note**: Fields `dest` and `owner` should have the same value.
+
+#### State format
+
+`id -> {encode(data, creators), tx_hash, tx_timestamp }`
```

Comment:
Already commented in a different comment but this whole section is likely redundant now and needs to be thought of differently. See Cosmos Cash example.

Reply:
Cosmos Cash describes DIDDoc and doesn't tell about ways of storage. I suggest leaving it here because it's and ADR about transactions. Not only protocol.
But I think we can remove creators from value because `data` already contains this.